runtime/Makefile

```diff
@@ -40,5 +49,20 @@ r-wrap: r_wrap.c r_split.h
 
 r-unwrap: r_unwrap.c r_split.h
 	gcc ${CFLAGS} r_unwrap.c -o r_unwrap
+
+
+libdgsh_a_SOURCES = negotiate.c $(DGSH_ASSEMBLY_FILE)
+dgsh_tee_SOURCES = dgsh-tee.c
+
+dgsh-tee:
+	git clone https://github.com/tammam1998/dgsh.git;\
```

Comment:
Would it make sense to fold this dependency under pash/runtime? (cc @angelhof )

Reply:
Do you mean using a submodule? The reason why @tammam1998 did it like this (correct me if I am wrong) is that dgsh tee is a very small part of dgsh and so this way it is just cloned, and then everything that is not necessary is deleted.

Reply:
sounds good, thanks!